### DISCUSSION TITLE
Passing options to autoprefix

### DISCUSSION THREAD

**Discussion by EML-github** [postcss/postcss#1905] [Q&A] [answered]
Sorry, basic question: how do I pass an option to autoprefixer (eg. 'cascade:false') when running postcss as follows?

 $ postcss custom.css --use autoprefixer -d build/

**Comment by ai** [comment, marked as answer]
The best way is to create PostCSS config.

But if you need CLI-only you can ask here https://github.com/postcss/postcss-cli

**Reply by EML-github** [reply]
Thanks Andrey - the config below does the job:

```
module.exports = {
   plugins: [ require('autoprefixer')({ cascade: false }) ]
}
```
There's no 'Discussions' page on postcss-cli, unfortunately.
